src/libxtp/gwbse/rpa.cc

```diff
@@ -91,6 +91,46 @@ Eigen::MatrixXd RPA::calculate_epsilon(double frequency) const {
 template Eigen::MatrixXd RPA::calculate_epsilon<true>(double frequency) const;
 template Eigen::MatrixXd RPA::calculate_epsilon<false>(double frequency) const;
 
+Eigen::MatrixXcd RPA::calculate_epsilon_complex(double frequency_r,double frequency_i) const {
```

Comment:
Use a `std::complex<double>` instead of two `double`

Reply:
Do you want to have just one complex number i.e
`Eigen::MatrixXcd RPA::calculate_epsilon_complex(std::complex<double> frequency) `
or do you want simply that double ---> std:complex \<double\>
` Eigen::MatrixXcd RPA::calculate_epsilon_complex(std::complex<double>double frequency_r,std::complex<double> frequency_i)  `

Reply:
one complex number

Reply:
Done